corekit/src/main/java/com/midtrans/sdk/corekit/core/merchant/MerchantApiManager.java

```diff
@@ -51,4 +59,117 @@ public void onFailure(@NonNull Call<CheckoutWithTransactionResponse> call, @NonN
             }
         }
     }
+
+    /**
+     * this method is used to get saved card list on merchant server
```

Comment:
@suryagsaputra please consult this with @jukiginanjar as he mentioned that there are changes regarding the get card list

Reply:
Completed. Updated.